pages/index.js

```diff
@@ -104,10 +90,10 @@ function HomePage() {
           <div className="py-2 lg:py-12">
             <Container wide>
               <div className="lg:flex">
-                <div className="mt-10 lg:mt-0 lg:flex-1">
+                <div className="mt-10 lg:mt-0 lg:w-1/2">
```

Comment:
@magicmatatjahu @mcturco Kindly look into this implementation. I have changed a little bit to make the spacings and components more uniform.